example_encryted_file/lib/main.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:io';
+import 'dart:typed_data';
+import 'package:http/http.dart' as http;
+
+import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
+import 'package:path_provider/path_provider.dart';
+
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  final response = await http.get(Uri.parse('https://images.pexels.com/photos/326055/pexels-photo-326055.jpeg?cs=srgb&dl=pexels-pixabay-326055.jpg&fm=jpg&w=1920&h=1080'));
```

Comment:
Why not store file somewhere in `assets`?

Reply:
it's for testing with real image. i prefer links because it's more flexible.

Reply:
hmm

Reply:
https://stackoverflow.com/questions/49480080/flutter-load-assets-for-tests